### ISSUE TITLE
[FR] Random delay

### ISSUE THREAD

**Issue by dokie80** [Nain57/Smart-AutoClicker#26]
First of all.. Thanks for the app. 
Is it possible to make delay time to be random.
For example if I put 20000 (20s), it will randomize between 18-22s..

Thanks in advance

**Comment by TREYWANGCQU** [comment]
Hello, it could be easily to add extra time to ```mainHandler.postDelayed``` like,
```java
    @WorkerThread
    private fun notifyClickDetection(click: ClickInfo) {
        isAreaFound = true

        mainHandler.post {
            detectionInfo?.second?.invoke(click)
        }
        val totalTime: Long = click.delayAfterMs+ Random.nextLong(0,300)  /** 😏😅*/
        mainHandler.postDelayed({
            isAreaFound = false
        }, totalTime)
    }
```
The file is named as [ScreenDetector.kt](https://github.com/Nain57/Smart-AutoClicker/blob/d3073f8519b89ec87968973003b2bed26e8dde12/detection/src/main/java/com/buzbuz/smartautoclicker/detection/ScreenDetector.kt)

**Comment by Nain57** [comment]
This will work, but only as a workaround as it will be applied to all clicks, regardless on the configuration.

The next millstone for Smart AutoClicker is the version 1.2, which will refactor the scenario management to provide several actions for the same condition.
Those actions will be of types CLICK/SWIPE/PAUSE, and each one will provide new options such as the random delay for a PAUSE.

Those behavioral changes requires a complete database overhaul, as well as a new UI to use it. I have just started this refactor (1.1 have just been released), but it should allow all required changes on the scenario management.
